cli/injector/main.go

```diff
@@ -271,6 +275,73 @@ func initExitSignalsHandler() {
 	signal.Notify(signals, syscall.SIGINT, syscall.SIGTERM)
 }
 
+// inject inject all the disruptions using the list of injectors
+func inject(kind string, sendToMetrics bool) (int, string) {
```

Comment:
We are already logging errors in this function, can we just return a boolean to tell if the injection (or cleanup) was successful instead?
```suggestion
# returns true if injection succeeded, false otherwise
func inject(kind string, sendToMetrics bool) bool {
```